### PR TITLE
Add Firestore query tombstone snippets

### PR DESCRIPTION
**Pull request by sorced-jim**
## Description

Add query tombstone snippets to be used https://cloud.google.com/fire…store/docs/best-practices#avoid_skipping_over_deleted_data

### CODE REVIEW

**Comment by dandhlee** [comment]
Hey @sorced-jim, could you address the lint errors? Give me a ping internally if you can't see the Kokoro logs.

File: firestore/cloud-client/snippets.py [hunk 1]
```diff
@@ -1009,3 +1009,59 @@ def create_and_build_bundle():
     # [END firestore_create_and_build_bundle]
 
     return bundle, bundle_buffer
+
+
+def _setup_tombstone(db):
+  now = datatime.datetime.now(tz=datetime.timezone.utc)
```
Comment (dandhlee) [inline firestore/cloud-client/snippets.py:1015]:
```suggestion
  now = datetime.datetime.now(tz=datetime.timezone.utc)
```

**Comment by kweinmeister** [comment]
@sorced-jim Do you still want to move forward with this PR?

**Comment by engelke** [comment]
Due to lack of responses and age, I'm closing this PR unmerged. Please reopen or recreate the PR if you still want it to be used.

**Pull request closed without merging**